app/models/epic_user.rb

```diff
@@ -35,19 +35,33 @@ def self.collection_name
 
   def self.for_identity(identity)
     begin
-      get(:viewuser, userid: identity.ldap_uid.split('@').first)
+      @result = get(:viewuser, userid: identity.ldap_uid.split('@').first)
+      @result["Silly_Name"] = @result.delete("UserID")
+
+      unless @result.present? && @result.is_a?(Hash) and @result.keys == ["UserID", "UserName", "IsExist", "IsActive", "IsBlocked", "IsPasswordChangeRequired", "IsSER"]
+        raise StandardError.new I18n.t("activerecord.errors.models.epic_user.attributes.base.epic_user_api_down")
+      else
+        return @result
+      end
     rescue => e
-      epic_error_webhook = Setting.get_value("epic_user_api_error_slack_webhook")
-
-      if epic_error_webhook.present?
-        notifier = Slack::Notifier.new(epic_error_webhook)
-        message = I18n.t('notifier.epic_user_api_slack_error', env: Rails.env)
-        message += "\n```#{e.class}\n"
-        message += "#{e.message}\n"
-        message += "#{e.backtrace[0..5]}```"
+      slack_epic_error_webhook = Setting.get_value("epic_user_api_error_slack_webhook")
+      teams_epic_error_webhook = Setting.get_value("epic_user_api_error_teams_webhook")
+
+      message = I18n.t('notifier.epic_user_api_slack_error', env: Rails.env)
+      message += "\n```#{e.class}\n"
```

Comment:
Does the same formatting work for Teams and Slack?

Reply:
It seems to, yes.